lib/logzio-nodejs.js

```diff
@@ -89,9 +98,28 @@ LogzioLogger.prototype._timerSend = function() {
     );
 };
 
+LogzioLogger.prototype._sendMessagesUDP = function() {
+    var messagesLength = this.messages.length;
+
+    for (var i=0; i<messagesLength; i++) {
+
+        var msg = this.messages[i];
+        msg.token = this.token;
+        var buff = new Buffer(stringifySafe(msg));
+
+        this.udpClient.send(buff, 0, buff.length, this.port, this.host, function(err, bytes) {
+            if (err) 
+                throw new Error('Failed to send udp log message. err = ' + err);
```

Comment:
Throwing an exception might the application which uses you. Let's use the same methodology below and call the callback with the error message and exception